packages/fcl/src/discovery/services.js

```diff
@@ -2,12 +2,9 @@ import {config} from "@onflow/config"
 import {invariant} from "@onflow/util-invariant"
 import {VERSION} from "../VERSION"
 
-const asyncPipe =
-  (...fns) =>
-  input =>
-    fns.reduce((chain, fn) => chain.then(fn), Promise.resolve(input))
+const isWindow = () => typeof window !== "undefined"
 
-async function addServices(services = []) {
+export async function getServices({ type }) {
```

Comment:
could this be an array for possibility of multiple service types being fetched? 
getServices({ types })

Reply:
@gregsantos Like that idea. Will update.